appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/OpenApiService.java

```diff
@@ -288,6 +289,7 @@ private OpenAPI buildDocument() throws OpenAPIBuildException {
                 openapi = new ApplicationProcessor(classes).process(openapi, appConfig);
                 openapi = new BaseProcessor(baseURLs).process(openapi, appConfig);
                 openapi = new FilterProcessor().process(openapi, appConfig);
+                openapi = new PathsProcessor().process(openapi, appConfig);
```

Comment:
The extra processor here adds unneeded technical overhead in my opinion. I would personally add this functionality to the BaseProcessor class.